src/design-system/styles/layout/grid-annotate.scss

```diff
@@ -6,7 +6,7 @@
 [class^="govuk-grid-column"] > p {
   @extend .govuk-body;
   display: block;
-  @include govuk-responsive-padding($govuk-spacing-responsive-6);
+  @include @include govuk-responsive-padding(6);
```

Comment:
Double `@include`

Reply:
fixed in later commit (that i didnt fixup yet)...